synapse/tests/test_cortex.py

```diff
@@ -1415,3 +1415,18 @@ def test_cortex_bytype(self):
         with s_cortex.openurl('ram:///') as core:
             core.formTufoByProp('inet:dns:a','woot.com/1.2.3.4')
             self.eq( len( core.eval('inet:ipv4*type="1.2.3.4"')), 2 )
+
+    def test_cortex_seq(self):
```

Comment:
What happens when a width is reached?
Does it roll over? Does it throw an exception?  That is not tested here.